ICSharpCode.Decompiler.Tests/TestCases/Pretty/GloballyQualifiedTypeInStringInterpolation.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ICSharpCode.Decompiler.Tests.TestCases.Pretty
+{
+	public static class GloballyQualifiedTypeInStringInterpolation
+	{
+		public static string Root => $"Prefix {(global::System.DateTime.Now)} suffix";
+		public static string Cast => $"Prefix {((int)global::System.DateTime.Now.Ticks)} suffix";
+#if CS100 && !NET40
```

Comment:
Should this have been `#if CS100 && NET60` instead?

Reply:
I think so.

Reply:
I think it makes more sense to use `!NET40` because classic framework is the "legacy edge case" and we want to minimize necessary changes when updating the framework version.

Reply:
This happens because `DefaultInterpolatedStringHandler` is only available on .NET 6 and above. If any of our tests ran on .NET 5, they would fail.

Reply:
Do you want me to change it back?

Reply:
Hmmm... yes, and because we emit conditional compilation symbols for all subsumed versions used, it works. OK, we can leave it like this...